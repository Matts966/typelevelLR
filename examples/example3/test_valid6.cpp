
#include <iostream>
#include "example3.hpp"
using namespace example3;

int main() {
  // 7 = 0b7
  std::cout << *begin()->O()->O()->O()->O()->O()->I()->I()->I()->end() << std::endl;
  return 0;
}
