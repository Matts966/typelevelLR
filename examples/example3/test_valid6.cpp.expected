RuleS0(Rule00(Rule00(Rule00(Rule00(Rule01(Rule11(Rule31(Accept()))))))))